runtime/oti/vmconstantpool.xml

```diff
@@ -176,6 +176,7 @@
 	<fieldref class="java/lang/StackWalker$StackFrameImpl" name="classLoaderName" signature="Ljava/lang/String;" jcl="se9_before_b165,se9"/>
 	<fieldref class="java/lang/StackWalker$StackFrameImpl" name="lineNumber" signature="I" jcl="se9_before_b165,se9"/>
 	<fieldref class="java/lang/StackWalker$StackFrameImpl" name="methodName" signature="Ljava/lang/String;" jcl="se9_before_b165,se9"/>
+	<fieldref class="java/lang/StackWalker$StackFrameImpl" name="methodSignature" signature="Ljava/lang/String;" jcl="se9_before_b165,se9"/>
```

Comment:
technically se9_before_b165 isn't required here, although it doesn't hurt and se9_before_b165 is obsolete anyway